indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +45,16 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
+        self, 
+        path_to_dir,
+        destination_dir,
+        accepted_types: Tuple[str] = ("pdf", "tiff", "tif", "doc", "docx")
+    ):
+        self.get_file_paths_from_dir(path_to_dir, accepted_types, True)
+        for file in self.file_paths:
+            filename = os.path.basename(file)
+            os.rename((file), f'{destination_dir}/{filename}')
```

Comment:
Why is `file` in parentheses? You should do some refactoring here. Should read something like: `os.rename(input_filepath, new_filepath)` for clarity

Reply:
Also, are there any potential errors that can get thrown here? Might want to have exceptions for those events.